openfisca_us/variables/hhs/medicaid/medicaid.py

```diff
@@ -79,6 +79,47 @@ def formula(person, period, parameters):
         return income_share_of_fpg <= fpg_income_threshold
 
 
+class pregnancy_medicaid_eligible(Variable):
```

Comment:
Could you also move this into its own file `variables/hhs/medicaid/pregnancy/is_pregnancy_medicaid_eligible.py`?

Similarly move the unit test file.
```suggestion
class is_pregnancy_medicaid_eligible(Variable):
```